crates/cubecl-macros/src/parse/expression.rs

```diff
@@ -118,11 +118,8 @@ impl Expression {
                     && args.iter().all(|arg| arg.is_const())
                     && method.method != "runtime"
                 {
-                    let receiver = receiver.as_const(context).unwrap();
-                    let method = &method.method;
-                    let args = args.iter().map(|it| it.to_tokens(context));
                     Expression::Verbatim {
-                        tokens: quote![#receiver.#method(#(#args),*)],
+                        tokens: quote![#method],
```

Comment:
Not sure why you can safely simplify this

Reply:
It was actually a bug, when the arguments as well as the method are const, we need to call the normal method with the normal arguments, not mix both (`sequence.index(u32)`, not `sequence.index(ExpandedElemTyped::from_lit(u32)` nor `sequence.__index_expand(...)`).